README.md

```diff
@@ -45,42 +45,43 @@ $handler = new WhatFailureGroupHandler(
 
 ### Configure LokiHandler Service
 ```yaml
-Itspire\MonologLoki\Handler\LokiHandler:
-  arguments:
-    $apiConfig:
-      entrypoint: "http://loki:3100"
-      context:
-        app: My-app
-      labels:
-        env: "%env(APP_ENV)%"
-      client_name: my_app_server
-      auth:
-        basic:
-          user: username
-          password: password
+  Itspire\MonologLoki\Handler\LokiHandler:
```

Comment:
Looks like this entire block was indented, when it wasn't necessary

Reply:
I think you accidentally looked at the old commit, I have reverted all my formatting changes to the README in the new version.